src/sonic-ctrmgrd-rs/crates/docker-rs/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+[package]
+name = "docker-rs"
```

Comment:
Non-blocking comment: It may make sense in the future to move this up to the "parent", if there aren't going to be any other components here.

Reply:
I anticipate there will be multiple apps in sonic-ctrmgrd-rs, so multiple crates.